plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/create-simple-product-block-editor.spec.js

```diff
@@ -0,0 +1,178 @@
+const { test, expect } = require( '@playwright/test' );
+
+const {
+	clickOnTab,
+	isBlockProductEditorEnabled,
+	toggleBlockProductEditor,
+} = require( '../../../../utils/simple-products' );
+
+const NEW_EDITOR_ADD_PRODUCT_URL =
+	'wp-admin//admin.php?page=wc-admin&path=%2Fadd-product';
+
+let isNewProductEditorEnabled = false;
+
+const isTrackingSupposedToBeEnabled = !! process.env.ENABLE_TRACKING;
+
+const productData = {
+	name: `Simple product Name ${ new Date().getTime().toString() }`,
+	summary: 'This is a product summary',
+	productPrice: '100',
+	salePrice: '90',
+};
+
+test.describe( 'General tab', () => {
+	test.describe.serial( 'Create product', () => {
+		let productId;
+		test.use( { storageState: process.env.ADMINSTATE } );
+
+		test.skip(
+			isTrackingSupposedToBeEnabled,
+			'The block product editor is not being tested'
+		);
+
+		test.beforeEach( async ( { browser } ) => {
+			const context = await browser.newContext();
+			const page = await context.newPage();
+			isNewProductEditorEnabled = await isBlockProductEditorEnabled(
+				page
+			);
+			if ( ! isNewProductEditorEnabled ) {
+				await toggleBlockProductEditor( 'enable', page );
+			}
+		} );
+
+		test.afterEach( async ( { browser } ) => {
+			const context = await browser.newContext();
+			const page = await context.newPage();
+			isNewProductEditorEnabled = await isBlockProductEditorEnabled(
+				page
+			);
+			if ( isNewProductEditorEnabled ) {
+				await toggleBlockProductEditor( 'disable', page );
+			}
+		} );
+
+		test( 'can create a simple product', async ( { page } ) => {
+			await page.goto( NEW_EDITOR_ADD_PRODUCT_URL );
+			await clickOnTab( 'General', page );
+			await page
+				.locator( '//input[@placeholder="e.g. 12 oz Coffee Mug"]' )
```

Comment:
Might want to consider using `.getByPlaceholder` instead of this locator.  It's a little more readable, and considered best practice in the Playwright community. https://playwright.dev/docs/api/class-framelocator#frame-locator-get-by-placeholder

Reply:
Good idea! I changed it in the commit [6de25cd](https://github.com/woocommerce/woocommerce/pull/39547/commits/6de25cd85faff32766a9d3624f0b4855e26c2ca0)